wdl/transforms/biscayne/src/main/scala/wdl/transforms/biscayne/ast2wdlom/AstToNewExpressionElements.scala

```diff
@@ -0,0 +1,14 @@
+package wdl.transforms.biscayne.ast2wdlom
+
+import common.validation.ErrorOr.ErrorOr
+import wdl.model.draft3.elements.ExpressionElement
```

Comment:
Is it not potentially problematic to be importing older models into the latest WDL version?

(I ask solely on the basis of pattern recognition after attempting to jettison as many draft-2 imports as possible from the 1.0 upgrade script.)

Reply:
Because right now the model is shared, I didn't do the package move out of `draft-3` into `base`, although for sanity I probably should just migrate everything in there en-masse.

But for now `model.draft3` is essentially `model.base`